src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -190,6 +209,11 @@ HttpStatusCode mongoSubscribeContext
 
     LM_T(LmtSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
 
+    //
+    // Any Scope of type SCOPE_TYPE_SIMPLE_QUERY in restriction.scopeVector?
```

Comment:
Probably a leftover, yes.
I moved this part to before the call to ... processAttributeVector?